lua/ulx/modules/sh/cfc_crucify.lua

```diff
@@ -0,0 +1,146 @@
+CFCUlxCommands.crucify = CFCUlxCommands.crucify or {}
+local cmd = CFCUlxCommands.crucify
+
+CATEGORY_NAME = "Fun"
+
+function cmd.uncrucifyPlayer( ply )
+    if not IsValid( ply ) then return "Invalid player!" end
+
+    local cross = ply.cross
+    ply.cross = nil
+
+    local ragdoll = ply.ragdoll
+    if IsValid( ragdoll ) then
+        ulx.unragdollPlayer( ply )
+    end
+
+    if IsValid( cross ) then

```

Comment:
You can use [`SafeRemoveEntity` ](https://wiki.facepunch.com/gmod/Global.SafeRemoveEntity) here